src/pages/ProjectSettings.tsx

```diff
@@ -30,10 +30,10 @@ import { getTabTitle } from "./projectSettings/getTabTitle";
 import { ProjectSettingsTabs } from "./projectSettings/Tabs";
 import { ProjectType } from "./projectSettings/tabs/utils";
 
-const { isProduction } = environmentalVariables;
+const { isDevelopment } = environmentalVariables;
 const { validateObjectId } = validators;
 
-const disablePage = isProduction();
+const disablePage = !isDevelopment;
```

Comment:
Since isDevelopment is a function I think `disablePage` would always evaluate to `false` which we're not ready for just yet!
```suggestion
const disablePage = !isDevelopment();
```